packages/core/src/r5/bundle-executor.ts

```diff
@@ -417,6 +417,7 @@ export class BundleExecutor {
     if (this.request.type === "transaction") {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       this.response = await this.client.transaction(this.request as any);
+      return;
```

Comment:
Nice fix :smile:.

Shoes this change also be made to the r4 `bundle-executor` as well? Also, is there any chance of adding a test for this?

Reply:
done!